l2tdevtools/dependency_writers/gift_ppa.py

```diff
@@ -131,3 +139,45 @@ def Write(self):
 
     with open(self.PATH, 'wb') as file_object:
       file_object.write(file_content)
+
+  def Write(self):
+    """Writes a gift_ppa_install.sh file."""
```

Comment:
Why use `raise NotImplementedError` and not `abc.abstractmethod` instead?

Reply:
To save importing all of abc: https://docs.python.org/2/library/exceptions.html#exceptions.NotImplementedError

Changed this anyway

Reply:
thx, seeing we use the abc approach more widely and sporadically NotImplementedError when necessary